### ISSUE TITLE
Accuracy By Category

### ISSUE THREAD

**Issue by Novicemonk** [kratzert/finetune_alexnet_with_tensorflow#71]
Hi Frederic, 
Great repo & support. Is there an easy way to determine:
1)  accuracy by category, and 
2) misclassified images
To improve my predictions, I'm trying to decide whether to clean my data or increase the number of images.  Apologies in advance if you addressed this in an earlier post, but searched through the closed posts & haven't been able to find anything. Have been having trouble getting Tensorboard up & fully running, if that's the solution.  Thanks!

**Comment by kratzert** [comment]
Hi @Novicemonk,
well this is nothing related directly to this code repo, so I never mentioned it anywhere. However, both should not be hard to get, or? When you do inference you have predicted label and true label for each image in the validation set. Then you can group by true label and calculate the accuracy for each true class. Further more, once you have the the predicted class for an image just compare against the true label, if it's wrong you store the image path. Both should be straight forward, even with the code provided in e.g. the notebook
